src/startup.rs

```diff
@@ -22,11 +24,29 @@ use crate::{
 #[derive(Clone)]
 pub struct HmacSecret(pub Secret<String>);
 
-pub struct AppState {
+#[derive(Clone)]
+pub struct AppState(Arc<InnerState>);
```

Comment:
Had to update our shared app state using the new type pattern because the library were using for signed cookies needs access to a `Key` that is a part of shared state